contracts/examples/lottery-egld/interaction/playground.ts

```diff
@@ -0,0 +1,93 @@
+import path = require("path");
```

Comment:
Is the file extension supposed to be .ts? This looks like Rust code (.rs).

Reply:
Nice that is looks like Rust :)

TypeScript :heart:

Reply:
My bad. Carry on. :+1: